examples/content_reuse/inputs/credentials.ini

```diff
@@ -1,21 +1,13 @@
-[google application credentials]
-GOOGLE_APPLICATION_CREDENTIALS = /***/google_cred.json
-
-[postman credentials]
-api_key =  ***Bearer key***
-postman_token = ***postman token***
-
-[redis]
-host=localhost
-port=6379
-password=
-
-
-[tagme credentials]
-gcube_token = ***
-postman_token = ***
-
-
-[kafka]
-host=localhost
-port=9092
\ No newline at end of file
+[graph]
```

Comment:
Maybe rename the file to something more appropriate. It's not just credentials we're passing here.